### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by parthdani-clevertap**
CleverTap Source documentation

### CODE REVIEW

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md [hunk 1]
```diff
@@ -1,4 +1,91 @@
...
+
+[CleverTap](https://clevertap.com/){:target="_blank”} is the all-in-one customer engagement platform that helps brands personalize and optimize all consumer touch points to improve user engagement, retention, and lifetime value.
+
+Using CleverTap as a source on Segment, you can evolve your team's engagement and marketing efforts. With CleverTap Exports, you can automatically forward your users' engagement and activity events to Segment. This will enable you to forward these events to your destinations or warehouses.
+
+Segment allows you to move CleverTap data to your favorite BI or analytics tool in a seamless manner using which you can combine CleverTap data with the event data already flowing into Segment. This will help you get a holistic report of your engagement and marketing efforts.
+
+This is an [Event Cloud Source](/docs/sources/#event-cloud-sources) which can export data into your Segment warehouse and federate the exported data into your enabled Segment destinations.
+
+This source is maintained by CleverTap. For any issues with the source, [contact their Support team]().
```
[the remark this answers is not in the document]
Comment (parthdani-clevertap) [reply src/connections/sources/catalog/cloud-apps/clevertap/index.md:14]: Link: https://help.clevertap.com/hc/en-us/requests/new

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md [hunk 2]
```diff
@@ -1,4 +1,91 @@
...
+| CT Source | Source on which this event was tracked, for example -SDK, API. The source is tracked with all events. |
+| Campaign Id | Id of the campaign associated with this event. |
+| Campaign name | Represents the name of the campaign, the message was part of. |
+| Campaign type | Represents the type of campaign, the message was part of. |
+| wzrk_pivot | Tracked when a campaign is sent using A/B Testing. Represents the variation. |
+| wzrk_c2a | Indicates the value of the button clicked by the user. This button can be present for the following campaign types: In-App, Push, or Mobile In-Box. |
+| Journey Id | Helps uniquely identify the journey in which the campaign is present. |
+| Campaign labels | Represent the labels added for the campaign. |
+
+You can refer to CleverTap's [Derived Events & Attributes](https://docs.clevertap.com/docs/export-format){:target="_blank”} to determine which attributes CleverTap forward to Segment.
```
[the remark this answers is not in the document]
Comment (parthdani-clevertap) [reply src/connections/sources/catalog/cloud-apps/clevertap/index.md:77]: This is sorted now. We have made this document live today.

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md [hunk 3]
```diff
@@ -1,4 +1,91 @@
...
+    - Note: Cross-region data is not allowed, so your events will be dropped by Segment if the region you select on the CleverTap is different from your Segment Workspace Region.
+8. Go back to Segment and navigate to your CleverTap source. Click **Add Destinations** to add any destinations that you want to receive CleverTap data.
+
+## Events
+
+The table below lists events that CleverTap sends to Segment. These events appear as tables in your warehouse, and as regular events in other Destinations.
+
+| Name | Description
+| --------------- | ----------- | ------------------------- |
+| App Launched | This event is raised every time a user launches the application. |
```
[the remark this answers is not in the document]
Comment (parthdani-clevertap) [reply src/connections/sources/catalog/cloud-apps/clevertap/index.md:38]: raised means triggered(captured) when user launches the application

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (forstisabella) [reply src/connections/sources/catalog/cloud-apps/clevertap/index.md:77]: Okay perfect, thanks for the update!

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md [hunk 4]
```diff
@@ -1,4 +1,91 @@
...
+| CT Source | Source on which this event was tracked, for example -SDK, API. The source is tracked with all events. |
+| Campaign Id | Id of the campaign associated with this event. |
+| Campaign name | Represents the name of the campaign, the message was part of. |
+| Campaign type | Represents the type of campaign, the message was part of. |
+| wzrk_pivot | Tracked when a campaign is sent using A/B Testing. Represents the variation. |
+| wzrk_c2a | Indicates the value of the button clicked by the user. This button can be present for the following campaign types: In-App, Push, or Mobile In-Box. |
+| Journey Id | Helps uniquely identify the journey in which the campaign is present. |
+| Campaign labels | Represent the labels added for the campaign. |
+
+You can refer to CleverTap's [Derived Events & Attributes](https://docs.clevertap.com/docs/export-format){:target="_blank”} to determine which attributes CleverTap forward to Segment.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/clevertap/index.md:77]:
```suggestion
You can refer to CleverTap's [Derived Events & Attributes](https://docs.clevertap.com/docs/export-format){:target="_blank”} documentation to determine which attributes CleverTap forward to Segment.
```

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md [hunk 5]
```diff
@@ -1,4 +1,91 @@
...
+## CleverTap Event Properties
+
+The table below list the properties included in the events listed above.
+
+| Name | Description |
+| ---- | ----------- |
+| CT App Version | Mobile Application version on which this event was tracked. App Version is tracked with all events. |
+| CT SDK Version | CleverTap SDK version on which this event was tracked. SDK Version is tracked with all events. |
+| CT Source | Source on which this event was tracked, for example -SDK, API. The source is tracked with all events. |
+| Campaign Id | Id of the campaign associated with this event. |
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/clevertap/index.md:69]:
```suggestion
| Campaign Id | ID of the campaign associated with this event. |
```

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md [hunk 6]
```diff
@@ -1,4 +1,91 @@
...
+| Campaign name | Represents the name of the campaign, the message was part of. |
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/clevertap/index.md:70]:
```suggestion
| Campaign name | Represents the name of the campaign, the message was part of. |
```
```suggestion
| Campaign name | The name of the campaign that the message was part of. |
```

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md [hunk 7]
```diff
@@ -1,4 +1,91 @@
...
+| Campaign type | Represents the type of campaign, the message was part of. |
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/clevertap/index.md:71]:
```suggestion
| Campaign type | The type of campaign that the message was part of. |
```

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md [hunk 8]
```diff
@@ -1,4 +1,91 @@
...
+| wzrk_pivot | Tracked when a campaign is sent using A/B Testing. Represents the variation. |
+| wzrk_c2a | Indicates the value of the button clicked by the user. This button can be present for the following campaign types: In-App, Push, or Mobile In-Box. |
+| Journey Id | Helps uniquely identify the journey in which the campaign is present. |
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/clevertap/index.md:74]:
```suggestion
| Journey Id | Helps uniquely identify the journey where the campaign is present. |
```

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md [hunk 9]
```diff
@@ -1,4 +1,91 @@
...
+| Campaign labels | Represent the labels added for the campaign. |
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/clevertap/index.md:75]:
```suggestion
| Campaign labels | The labels added for the campaign. |
```

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (forstisabella) [reply src/connections/sources/catalog/cloud-apps/clevertap/index.md:38]: Let's replace "raised" with "captured" to fit more with the rest of the content on our docs site, if that's okay with you!

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md [hunk 10]
```diff
@@ -1,4 +1,91 @@
...
+    - **Note:** Cross-region data is not allowed. Your events will be dropped by Segment if the region you select in CleverTap is different from your Segment Workspace Region.
+8. Go back to Segment and navigate to your CleverTap source. Click **Add Destinations** to add any destinations that you want to receive CleverTap data.
+
+## Events
+
+The table below lists events that CleverTap sends to Segment. These events appear as tables in your warehouse and as regular events in other Destinations.
+
+| Name | Description
+| --------------- | ----------- | ------------------------- |
+| App Launched | This event is raised every time a user launches the application. |
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/clevertap/index.md:38]:
```suggestion
| App Launched | This event is captured every time a user launches the application. |
```

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md [hunk 11]
```diff
@@ -1,4 +1,91 @@
...
+| App Installed | The event is raised when the user launches the app for the first time. |
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/clevertap/index.md:39]:
```suggestion
| App Installed | The event is captured when the user launches the app for the first time. |
```

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md [hunk 12]
```diff
@@ -1,4 +1,91 @@
...
+| App Uninstalled | This event is recorded when a user uninstalls your application. |
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/clevertap/index.md:40]:
```suggestion
| App Uninstalled | This event is captured when a user uninstalls your application. |
```

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md [hunk 13]
```diff
@@ -1,4 +1,91 @@
...
+| App Version Changed | This event is raised when a user’s current app version is different from the user’s previous app version. |
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/clevertap/index.md:41]:
```suggestion
| App Version Changed | This event is captured when a user’s current app version is different from the user’s previous app version. |
```

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md [hunk 14]
```diff
@@ -1,4 +1,91 @@
...
+| Notification Sent | The event is tracked when the notification is successfully sent from CleverTap to the communication channel you select for your campaign. |
+| Notification Viewed | This event is tracked when a user views an email, in-app notification, or web notification sent from CleverTap. |
+| Notification Clicked | This event is tracked only when a user clicks on a notification sent via CleverTap.Recorded when a user clicks on a mobile push, in-app, email, web popup, or web push message sent via the CleverTap dashboard or through the campaign API.|
+| Push Impressions | This event is tracked when a Push notification sent from CleverTap is delivered on a user’s device. |
+| Notification Replied | This event is recorded when a user replies to a WhatsApp message. |
+| Control Group | The event is raised when a campaign is activated with a Control group. |
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/clevertap/index.md:47]:
```suggestion
| Control Group | The event is captured when a campaign is activated with a Control group. |
```

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md [hunk 15]
```diff
@@ -1,4 +1,91 @@
...
+| Channel Unsubscribed | The event is raised when a user Unsubscribes to receive further communication through a channel|
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/clevertap/index.md:48]:
```suggestion
| Channel Unsubscribed | The event is captured when a user Unsubscribes to receive further communication through a channel|
```

File: src/connections/sources/catalog/cloud-apps/clevertap/index.md [hunk 16]
```diff
@@ -1,4 +1,91 @@
...
+| Reply Sent | This event is recorded when an agent (CleverTap user) replies to a message from the end user.|
+| Webhook Delivered | This event is tracked when a Webhook is delivered from CleverTap |
+| UTM Visited | This event is tracked when a user clicks on a link from a marketing campaign that has a UTM parameter defined on it. |
+| AB Experiment Stopped | This event is recorded when the A/B experiment is stopped. |
+| AB Experiment Rolled Out | This event is recorded when the A/B experiment is started. |
+| AB Experiment Disqualified | This event is raised when the device is disqualified. |
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/clevertap/index.md:54]:
```suggestion
| AB Experiment Disqualified | This event is captured when the device is disqualified. |
```

**Review by forstisabella — APPROVED** [review]
a few small fixes and we'll be good to go!

**Comment by parthdani-clevertap** [comment]
[forstisabella](https://github.com/forstisabella) Any update on this? Let me know the next steps to make this documentation live as well as our Source live in Private beta.

**Pull request merged**